ROLES.md

```diff
@@ -188,18 +175,19 @@ The following apply to the part of the codebase for which one would be an approv
 
 ## Lead
 
-Working group leads, or just ‘leads’, are approvers of an entire area that have demonstrated good judgement and responsibility.
-Leads accept design proposals and approve design decisions for their area of ownership.
+Working group leads, or just ‘leads’, are maintainers of an entire area that have demonstrated good judgement and responsibility.
+Leads accept design proposals and approve design decisions for their area of ownership, triage issues, and run regular working group
+meetings.
 
 ### Requirements
 
 Getting to be a lead of an existing working group:
 
 * Recognized as having expertise in the group’s subject matter
 
-* Approver for some part of the codebase for at least 3 months
+* Maintainer for some part of the codebase for at least 3 months
 
-* Member for at least 1 year (relevant as Istio ages a bit more)
+* Member for at least 1 year
```

Comment:
Let's ask the TOC to focus on this one.  Is the time right for removing the relevance clause?